var/spack/repos/builtin/packages/vtk-m/package.py

```diff
@@ -165,7 +166,7 @@ class VtkM(CMakePackage, CudaPackage, ROCmPackage):
 
     # VTK-M PR#3258
     # https://gitlab.kitware.com/vtk/vtk-m/-/merge_requests/3258
-    patch("mr3258-fix-typo-thrust-dependency-with-rocm.patch", when="@2.2:")
+    patch("mr3258-fix-typo-thrust-dependency-with-rocm.patch", when="@2.2")
```

Comment:
I think this should actually be

```suggestion
    patch("mr3258-fix-typo-thrust-dependency-with-rocm.patch", when="@2.2.0")
```

I believe [MR3258](https://gitlab.kitware.com/vtk/vtk-m/-/merge_requests/3258) was backported to the 2.2 release branch. So, if there is ever a 2.2.1, it will also not need this patch.